test/specs/collections/Table/TableHeader-test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import * as common from 'test/specs/commonTests'
+
+import TableHeader from 'src/collections/Table/TableHeader'
+
+describe('TableHeader', () => {
+  common.isConformant(TableHeader)
+  common.rendersChildren(TableHeader)
+
+  it('renders as a thead by default', () => {
+    shallow(<TableHeader />)
+      .should.have.tagName('thead')
+  })
+
+  describe('fullWidth', () => {
```

Comment:
>As really solution I see updating all common tests to handle additional argument, but I don't like it because we have only two cases for this now.

Agreed, though we can easily default `className = propKey`. I've checked and there is only one test using `requiredProps`, so there is very little to update:


```jsx
export const propKeyOnlyToClassName = (Component, propKey, className = propKey, requiredProps = {}) => {}
```

I'll make a PR that does this, then it can be overridden to work with `full-width` and `single line`.